hw/bsp/family_support.cmake

```diff
@@ -36,12 +36,14 @@ if (NOT EXISTS ${CMAKE_CURRENT_LIST_DIR}/${FAMILY}/family.cmake)
   message(FATAL_ERROR "Family '${FAMILY}' is not known/supported")
 endif()
 
-# enable LTO if supported
-include(CheckIPOSupported)
-check_ipo_supported(RESULT IPO_SUPPORTED)
-if (IPO_SUPPORTED)
-  set(CMAKE_INTERPROCEDURAL_OPTIMIZATION TRUE)
-endif ()
+if (NOT TINYUSB_OPT_SKIP_CHECK_IPO_SUPPORTED)
```

Comment:
yeah this makes sense. Actualy I think we should skip this entirely for rp2040. LTO is global option, and it should only be set by top level application cmakelists.txt. It is ok for other ports since it only applies to examples within tinyusb. But I don't really want to force this on user application at all.